lib/modules/platform/gitlab/index.ts

```diff
@@ -538,9 +538,22 @@ async function ignoreApprovals(pr: number): Promise<void> {
       }[]
     >(url);
 
+    const ruleName = 'renovateIgnoreApprovals';
+
     const existingAnyApproverRule = rules?.find(
       ({ rule_type }) => rule_type === 'any_approver'
     );
+    const existingRegularApproverRules = rules?.filter(
+      ({ rule_type, name }) => rule_type !== 'any_approver' && name !== ruleName
+    );
+
+    if (existingRegularApproverRules.length) {
```

Comment:
```suggestion
    if (existingRegularApproverRules?.length) {
```
will fix cannot read length of undefined 🙃